src/main/java/org/sagebionetworks/web/client/widget/entity/file/BasicTitleBarViewImpl.java

```diff
@@ -21,7 +21,7 @@
 	@UiField
 	SimplePanel favoritePanel;
 	@UiField
-	Icon entityIcon;
+	EntityTypeIcon entityIcon;
```

Comment:
Before/After 
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/17580037/144890872-032b95d3-113f-4341-8430-59d5742dbe65.png">

Reply:
May want a bit more space between the large icon and the entity name.

Reply:
Done.